database/entity/0067-private_key_in_community_table/Community.ts

```diff
@@ -0,0 +1,63 @@
+import {
+  BaseEntity,
+  Entity,
+  PrimaryGeneratedColumn,
+  Column,
+  CreateDateColumn,
+  UpdateDateColumn,
+} from 'typeorm'
+
+@Entity('communities')
+export class Community extends BaseEntity {
+  @PrimaryGeneratedColumn('increment', { unsigned: true })
+  id: number
+
+  @Column({ name: 'foreign', type: 'bool', nullable: false, default: true })
+  foreign: boolean
+
+  @Column({ name: 'url', length: 255, nullable: false })
+  url: string
+
+  @Column({ name: 'public_key', type: 'binary', length: 64, nullable: false })
+  publicKey: Buffer
+
+  @Column({ name: 'private_key', type: 'binary', length: 64, nullable: true })
```

Comment:
why not setting the length to 32 directly for this new column, if you for yourself work on reducing the length for keypairs?

Reply:
The private key has a length of 64 bytes, the public key has 32 bytes.
This is the private key.

Reply:
This PR handles to correct the public_key size: https://github.com/gradido/gradido/pull/3026
It is unreleated to this PR

Reply:
Ok, i understand...